src/app/rosetta/lib/account.ml

```diff
@@ -282,8 +320,8 @@ module Balance = struct
       ; db_block_identifier_and_balance_info=
           (fun ~block_query ~address ->
             let (module Conn : Caqti_async.CONNECTION) = db in
-            Sql.run (module Conn) block_query address )
-      ; validate_network_choice= Network.Validate_choice.Real.validate }
+            Sql.run ~graphql_uri (module Conn) block_query address )
+      ; Validate_network_choice= Network.Validate_choice.Real.validate }
```

Comment:
Out of curiosity, why is `Validate_network_choice` with a capital V now?

Reply:
That's a good question. And even better one is why the heck does it compile!? :sweat_smile:
Fixing.